### DISCUSSION TITLE
请教如何清理数据库中的log信息，以及重置成败数

### DISCUSSION THREAD

**Discussion by mathlife** [qd-today/qd#239] [Q&A] [answered]
我使用sqlite viewer看了一下，数据库中log表内保留了不少已经不签到的任务的log信息，是否有办法清理。还有就是每个签到任务的成败数可以重置么？网上搜了清空数据库表的方法，结果直接提示database disk image is malformed无法使用了。感谢回复

**Comment by a76yyyy** [comment, marked as answer]
最新版本已经支持该功能的网页端操作了
